lib/listener/exit.js

```diff
@@ -23,4 +23,10 @@ module.exports = function () {
       process.exitCode = 1;
     }
   });
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4736995)